packages/replay-next/components/sources/log-point-panel/Capsule.tsx

```diff
@@ -104,20 +104,24 @@ export default function Capsule({
           data-test-state={tooManyPointsToFind ? "too-many-points" : "valid"}
           onClick={onClickFocusContentEditable}
         >
-          <NumberEditor
-            className={styles.CurrentIndex}
-            data-exact={currentHitPoint !== null || undefined}
-            data-test-name="LogPointCurrentStepInput"
-            data-too-many-points-to-find={tooManyPointsToFind || undefined}
-            data-value={closestHitPointIndex + 1}
-            defaultValue={closestHitPointIndex + 1}
-            disableSelectionWhenNotFocused
-            editable={!tooManyPointsToFind}
-            maxValue={Math.max(1, hitPoints.length)}
-            minValue={1}
-            onSave={onSave}
-            ref={editorRef}
-          />
+          {hitPoints.length === 0 ? (
+            <div className={styles.CurrentIndex}>0</div>
+          ) : (
+            <NumberEditor
+              className={styles.CurrentIndex}
+              data-exact={currentHitPoint !== null || undefined}
+              data-test-name="LogPointCurrentStepInput"
+              data-too-many-points-to-find={tooManyPointsToFind || undefined}
+              data-value={closestHitPointIndex + 1}
+              defaultValue={closestHitPointIndex + 1}
+              disableSelectionWhenNotFocused
+              editable={!tooManyPointsToFind}
+              maxValue={Math.max(1, hitPoints.length)}
```

Comment:
do we have to keep this `Math.max` here? 🤔

Reply:
Nah, it would be okay to remove it now. I had left it there because it wasn't hurting anything but I don't mind.